src/model/keychain.ts

```diff
@@ -30,13 +35,27 @@ interface AnonymousKeyData {
   [key: string]: AnonymousKey;
 }
 
+const saveStringMMKV = (key: string, value: string) => {
+  keychainLocalStorage.set(key, value);
+};
+
+const loadStringMMKV = (key: string) => {
+  return keychainLocalStorage.getString(key);
+};
+
 export async function saveString(
   key: string,
   value: string,
   accessControlOptions: Options
 ): Promise<void> {
   return new Promise(async (resolve, reject) => {
     try {
+      // if data is public, save to local storage
+      if (!accessControlOptions?.accessControl) {
```

Comment:
can we have something more explicitly defined for public vs non-public access control options (same for `saveObject`)

Reply:
i can add a separate fn prop if we want, this is why im able to do this currently 

public data access control options are 
`{
 accessible: ACCESSIBLE.WHEN_UNLOCKED_THIS_DEVICE_ONLY
 }`
 
 private data access control options are 
 ` {
 accessControl: ios
          ? ACCESS_CONTROL.USER_PRESENCE
          : ACCESS_CONTROL.BIOMETRY_CURRENT_SET_OR_DEVICE_PASSCODE,
 accessible: ACCESSIBLE.WHEN_UNLOCKED_THIS_DEVICE_ONLY
 }`